packages/desktop-lib/src/lib/plugin-system/events/plugin.event.ts

```
@@ -1,11 +1,12 @@
+import { ID, PluginOSArch, PluginOSType } from '@gauzy/contracts';
 import { logger } from '@gauzy/desktop-core';
 import { ipcMain, IpcMainEvent } from 'electron';
+import * as os from 'os';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)